network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/article_hero_guts.html

```diff
@@ -0,0 +1,90 @@
+{% load static i18n wagtailimages_tags homepage_tags%}
+{% get_page_authors page as authors %}
+
```

Comment:
This file follows the other article/publication file pretty closely with some additional if statements for whether or not the user would like to render some things such as the authors. However in preparation for the P2 asks which are custom colors and such, I think this is best in its own file so we can update it accordingly in the future without making any changes to the original article/publication header file